sql/hive-thriftserver/src/test/scala/org/apache/spark/sql/hive/thriftserver/ThriftServerWithSparkContextSuite.scala

```diff
@@ -207,6 +207,16 @@ trait ThriftServerWithSparkContextSuite extends SharedThriftServer {
       // scalastyle:on line.size.limit
     }
   }
+
+  test("SPARK-43119: Get SQL Keywords") {
+    withCLIServiceClient() { client =>
```

Comment:
As we deliver hive-jdbc v2.3.9 and hive-service-rpc v3.1.3, we can not test with JDBC directly as the lack of implementation at the client side